src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -36,14 +36,22 @@
                 (click)="reset()">
           <i [ngClass]="[icons.undo]"></i>
         </button>
+        <button class="btn btn-light my-1 ml-3"
+                i18n-title
+                title="Information"
+                (click)="showMessage = !showMessage">
+          <i [ngClass]="[icons.infoCircle, icons.large]"></i>
+        </button>
       </div>
     </div>
   </div>
 
   <div class="row">
-    <div class="col">
+    <div class="col"
+         *ngIf="!showMessage">
```

Comment:
Probably we don't need to hide the iframe... Just in case someone clicks there when Grafana is working for them.

Reply:
updated :)